### PR TITLE
Bring the README up-to-date with current standards

### PR DESCRIPTION
**Pull request by tech234a**

### CODE REVIEW

File: README.wget-at.md [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
+### General instructions
 
-In most of the below cases, there will be a web interface running at http://localhost:8001/. If you don't know or care what this is, you can just ignore it—otherwise, it gives you a fancy view of what's going on.
+Data integrity is very important in Archive Team projects. Please note the following important rules:
 
-**If anything goes wrong while running the commands below, please scroll down to the bottom of this page. There's troubleshooting information there.**
+* [Do not use proxies or VPNs](https://wiki.archiveteam.org/index.php/ArchiveTeam_Warrior#Can_I_use_whatever_internet_access_for_the_Warrior?).
+* Run the project using the either the Warrior or the project-specific Docker container as listed below. [Do not modify project code](https://wiki.archiveteam.org/index.php/ArchiveTeam_Warrior#I'd_like_to_help_write_code_or_I_want_to_tweak_the_scripts_to_run_to_my_liking._Where_can_I_find_more_info?_Where_is_the_source_code_and_repository?). Compiling the project dependencies yourself is no longer supported.
+* Only use your tracker nickname(s) on machines you personally operate. Nickname sharing makes it harder to inspect data if a problem arises.
```
Comment (TheTechRobo) [inline README.wget-at.md:15]: I'd clarify that it's fine to share nicknames between machines

File: README.wget-at.md [hunk 2]
```diff
@@ -30,197 +39,15 @@ after which the project can be run:
...
-
-    chmod o+rX -R /usr/local/lib/python2.7/dist-packages
-
-### run-pipeline: command not found
-
-Install `seesaw` using `pip2` instead of `pip`.
+Be sure to replace `YOURNICKHERE` with the nickname that you want to be shown as on the tracker. You don't need to register it, just pick a nickname you like.
 
-    pip2 install seesaw
+At this point, there will be a web interface running at http://localhost:8001/. If you don't know or care what this is, you can just ignore it—otherwise, it gives you a fancy view of what's going on.
```
Comment (TheTechRobo) [inline README.wget-at.md:44]: This doesn't happen in most Docker containers, IIRC.

**Review by TheTechRobo — CHANGES_REQUESTED** [review]
Just some minor nits. This looks good!

One thing, though: should the non-Wget+AT readmes be updated too?

File: README.wget-at.md (hunk 2 above)
Comment (tech234a) [reply README.wget-at.md:44]: Yeah it's usually disabled by default using command line arguments

File: README.wget-at.md (hunk 1 above)
Comment (tech234a) [reply README.wget-at.md]: I tried to indicate that already by using the word "machines" but perhaps that was unclear. I revised it a bit, though it does make the sentence a bit longer. I did want to try to keep these points somewhat brief to increase the chance that they would be read.

**Comment by tech234a** [comment]
> One thing, though: should the non-Wget+AT readmes be updated too?

According to @arkiver2 in #imgone IRC logs:
```
the README i keep 'reusing' is https://github.com/ArchiveTeam/standalone-readme-template/blob/master/README.wget-at.md . if there should be updates to the README, they should be made to that template
```

**Comment by rewbycraft** [comment]
This looks very nice!

**Comment by Arkiver2** [comment]
Thank you very much for this @tech234a !

**Pull request merged**